src/components/Popover/src/PopoverBubble.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div
+		:class="$s.PopoverBubble"
+		:style="computedStyles"
+	>
+		<!-- @slot Popover bubble content -->
+		<slot />
+	</div>
+</template>
+
+<script>
+import chroma from 'chroma-js';
+
+export default {
+	name: 'PopoverBubble',
+
+	props: {
```

Comment:
Rather than direct access to the popover container as an additional child component, I think we should have a single "Popover" component and route these props through the parent to this internal child.

We'll also need to add some "padding" option, more details below.